cmd/nodepool/azure/create.go

```diff
@@ -25,6 +25,7 @@ func NewCreateCommand(coreOpts *core.CreateNodePoolOptions) *cobra.Command {
 	}
 	cmd.Flags().StringVar(&o.instanceType, "instance-type", o.instanceType, "The instance type to use for the nodepool")
 	cmd.Flags().Int32Var(&o.diskSize, "root-disk-size", o.diskSize, "The size of the root disk for machines in the NodePool (minimum 16)")
+	cmd.Flags().StringVar(&o.availabilityZone, "availability-zone", o.availabilityZone, "The availabilityZone for the nodepool. Must be left unspecified if in a region that doesn't support AZs")
```

Comment:
can we use either the term "availability-zone" or "zone" consistently? In https://github.com/openshift/hypershift/pull/1127/files#diff-c33d1bcaa1a2d60feb64fd97f42a5d4ea7d822e9da901e8275d5865d945faa2bR34

Reply:
Updated to use `availability-zone` consistently